### DISCUSSION TITLE
External links

### DISCUSSION THREAD

**Discussion by vicnz** [vercel/next.js#22837] [Help] [answered]
I'm using a normal anchor tag
`<a href="www.google.com/">link</a>`
instead of directing to google.com it uses the server url
**example**
`localhost:3000/www.google.com`

can this be altered in the next.config.js?

**Comment by MahefaAbel** [comment, marked as answer]
Can't you add **http://** in the href ? Like this :
`<a href="http://www.google.com/">link</a>`

**Reply by vicnz** [reply]
Thanks it works, guess it's relying on the file:// protocol

**Reply by MahefaAbel** [reply]
Yes, you can mark this discussion resolved too ;)
